src/main/java/com/adyen/model/checkout/SubMerchant2.java

```diff
@@ -0,0 +1,254 @@
+/*
+ * Adyen Checkout API
+ *
+ * The version of the OpenAPI document: 70
+ * 
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+
+package com.adyen.model.checkout;
+
+import java.util.Objects;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.HashMap;
+import com.adyen.model.checkout.Address;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import com.fasterxml.jackson.annotation.JsonValue;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import com.fasterxml.jackson.core.JsonProcessingException;
+
+
+/**
+ * SubMerchant2
+ */
+@JsonPropertyOrder({
+  SubMerchant2.JSON_PROPERTY_ADDRESS,
+  SubMerchant2.JSON_PROPERTY_ID,
+  SubMerchant2.JSON_PROPERTY_MCC,
+  SubMerchant2.JSON_PROPERTY_NAME,
+  SubMerchant2.JSON_PROPERTY_TAX_ID
+})
+@JsonTypeName("SubMerchant-2")
+
+public class SubMerchant2 {
```

Comment:
is this another "2" case? should this be fixed on the specs first?

Reply:
This is an unfortunate case. I'm wondering whether it's worth it to go back and fix this right now.

Reply:
The submerchant2 class has a ID parameter and Address instead of city + country param. I propose we refactor the submerchant2 class to SubMerchantInfo manually in this library, and then change it later on main to SubMerchantInfo so we don't have any breaking changes.